go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/kubevirt/kubevirt-tekton-tasks
 
 go 1.21
-toolchain go1.22.2
+toolchain go1.22.3
```

Comment:
Because the previous PR was closed and as a daily reminder :)

@ksimon1 Can we update the project to go 1.22?

Reply:
The problem is, that d/s is still using Go 1.21